samples/rand_pfault_check.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * Wrappers to randomly touch pages.
```

Comment:
Extra description here would also help me to understand the motivations behind this patch, I would also recommend embedding some form of the commit message here too.